CHANGELOG.md

```diff
@@ -78,6 +79,67 @@ title: Changelog
+- each metric should have an expire time [#11838](https://github.com/apache/apisix/pull/11838)
+
+### Bugfixes
+
+- Fix: resync etcd when a lower revision is found [#12015](https://github.com/apache/apisix/pull/12015)
+- Fix: remove model options' `stream` default value [#12013](https://github.com/apache/apisix/pull/12013)
+- Fix: grpc-web response contains two trailer chunks [#11988](https://github.com/apache/apisix/pull/11988)
+- Fix: event_id is nil in chaitin-waf [#11651](https://github.com/apache/apisix/pull/11651)
+- Fix: race condition problem while update upstream.nodes [#11916](https://github.com/apache/apisix/pull/11916)
+- Fix: `upstream_obj.upstream` should not be a string [#11932](https://github.com/apache/apisix/pull/11932)
+- Fix: query params in override.endpoint not being sent to LLMs [#11863](https://github.com/apache/apisix/pull/11863)
+- Fix: corrupt data in routes() response due to healthchecker data [#11844](https://github.com/apache/apisix/pull/11844)
+- Fix: deepcopy should copy same table exactly only once [#11861](https://github.com/apache/apisix/pull/11861)
+- Fix: disallow empty key configuration attributes [#11852](https://github.com/apache/apisix/pull/11852)
+- Fix: etcd watch restart when receive invalid revision [#11833](https://github.com/apache/apisix/pull/11833)
+- Fix: missing parsed_url nil check [#11637](https://github.com/apache/apisix/pull/11637)
+- Fix: use `plugin.get` to fetch plugin configured in multi-auth plugin [#11794](https://github.com/apache/apisix/pull/11794)
+- Fix: allow special characters in uri params [#11788](https://github.com/apache/apisix/pull/11788)
+- Fix: add nil check to conf in body-transformer [#11768](https://github.com/apache/apisix/pull/11768)
+- Fix: use max_req_body_bytes field in custom_format [#11771](https://github.com/apache/apisix/pull/11771)
+- Fix: health checker can't be released due to health parent being released early [#11760](https://github.com/apache/apisix/pull/11760)
+- Fix: use right modifiedIndex for consumer when use credential [#11649](https://github.com/apache/apisix/pull/11649)
+
+### Core
+
+- upgrade openresty version to v1.27.11 [#11936](https://github.com/apache/apisix/pull/11936)
+- add support for "system" ssl_trusted_certificate [#11809](https://github.com/apache/apisix/pull/11809)
+- support _meta.pre_function to execute custom logic before execution of each phase [#11793](https://github.com/apache/apisix/pull/11793)
+- support anonymous consumer [#11917](https://github.com/apache/apisix/pull/11917)
+- accelerate the creation of the consumer cache [#11840](https://github.com/apache/apisix/pull/11840)
+- replace 'string.find' with 'core.string.find' [#11886](https://github.com/apache/apisix/pull/11886)
+- workflow plugin registration [#11832](https://github.com/apache/apisix/pull/11832)
+
+### Plugins
+
+- support embeddings API [#12062](https://github.com/apache/apisix/pull/12062)
```

Comment:
this PR should also be mentioned with `ai-proxy` CHANGE as well.

Reply:
okay